docs/guides/docs/running-a-node/index.mdx

```diff
@@ -31,6 +31,16 @@ Running your own node provides several advantages:
 1. **Query Freedom:** By hosting your own node, you can execute a higher number of queries without encountering any rate limits.
 2. **Network Independence:** Having your own node ensures that you're not reliant on third-party services, giving you full control over your interactions with the Fuel blockchain.
 
+## Minimum Requirements
```

Comment:
Are these really required to run a local node for just testing if your contract works without connecting to any network?

Reply:
This tool is designed for much more intensive use than just testing example contracts. While I agree most average users won't require it, teams that are processing a significant amount of information shouldn't be constrained by our API's limitations. For instance, the Fuelet team, which is developing their dashboard, will find it particularly useful.

Reply:
Can you add a note about this so that anyone who just wants to test their contract locally doesn't get scared away?

Reply:
Yea sure ill add a note :)

Reply:
Thanks! Can you move it so it's under the "Hardware Requirements" header?